src/common/__tests__/log-search-store.test.ts

```diff
@@ -0,0 +1,82 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+
+import { LogSearchStore } from "../../renderer/components/dock/log-search/store";
+
+const logs = [
+  "1:M 30 Oct 2020 16:17:41.553 # Connection with replica 172.17.0.12:6379 lost",
+  "1:M 30 Oct 2020 16:17:41.623 * Replica 172.17.0.12:6379 asks for synchronization",
+  "1:M 30 Oct 2020 16:17:41.623 * Starting Partial resynchronization request from 172.17.0.12:6379 accepted. Sending 0 bytes of backlog starting from offset 14407.",
+];
+
+describe("search store tests", () => {
+  let logSearchStore: LogSearchStore;
+
+  beforeEach(() => {
+    logSearchStore = new LogSearchStore();
+  });
+
+  it("does nothing with empty search query", () => {
+    logSearchStore.onSearch([], "");
```

Comment:
Calling `onSearch()` feels like stateful solution where there should be reactivity. Superficial.